docs/conf.py

```diff
@@ -313,3 +319,31 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 # texinfo_no_detailmenu = False
+
+link_files = {
+    f: dict(
+        using=dict(
+            GH='https://github.com',
+        ),
+        replace=[
+            dict(
+                pattern=r'((Issue|PR)\s?)?#(?P<issue_or_pr>\d+)',
+                url='{GH}/aio-libs/{project}/issues/{issue_or_pr}',
+            ),
+            dict(
+                pattern=r'^(?m)((?P<scm_version>v?\d+(\.\d+){1,2}))\n[-=]+\n',
```

Comment:
What this rule does?

Reply:
Oh, it's automatic replacement of version titles appending the release date from git tags.